### ISSUE TITLE
[BUG] The GUI fails to start, cannot import sip

### ISSUE THREAD

**Issue by emansije** [SamSchott/maestral-dropbox#74]
**Describe the bug**
When trying to start the GUI, Maestral throws an error:

```
maestral gui                     
Traceback (most recent call last):
  File "/home/user/.local/bin/maestral", line 8, in <module>
    sys.exit(main())
  File "/home/user/.local/lib/python3.7/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/home/user/.local/lib/python3.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/home/user/.local/lib/python3.7/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/user/.local/lib/python3.7/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/user/.local/lib/python3.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/home/user/.local/lib/python3.7/site-packages/maestral/cli.py", line 189, in gui
    from maestral.gui.main import run
  File "/home/user/.local/lib/python3.7/site-packages/maestral/gui/main.py", line 21, in <module>
    from PyQt5 import QtCore, QtWidgets, sip
ImportError: cannot import name 'sip' from 'PyQt5' (/usr/lib/python3/dist-packages/PyQt5/__init__.py)
```

**To Reproduce**
Typing `maestral gui` on the command line.

**Expected behaviour**
Starting the GUI.

**System (please complete the following information):**
 - Maestral version: 0.4.4
 - OS: Kubuntu 19.10 amd64
 - Desktop environment: KDE Plasma 5.16.5
 - Qt version: 5.12.3+dfsg-2 amd64

**Additional context**
Maestral was installed with `pip3 install --user --upgrade maestral`. Even after installing sip with `pip3 install --user --upgrade sip`, Maestral kept throwing the same error.

**Comment by SamSchott** [comment]
You should not need to install or upgrade `sip` manually, PyQt5 comes with its own internal version of it. However, it is this import which seems to fail: `from PyQt5 import sip`.

Did you install PyQt5 from PyPI or with your distro's package manager? In the first case, could you uninstall the separate version of `sip` and update PyQt5 itself? In the second case, I am not sure how to resolve it. But the part of Maestral which needs access to `sip` can be easily changed if need be, it is only a single function.

**Comment by emansije** [comment]
I installed PyQt5 with my distro's package manager. I tried to do `from PyQt5 import sip` on a python3 shell and it fails. I guess my distro's version of PyQt5 is outdated. :disappointed:

**Comment by SamSchott** [comment]
I don't think it's because your distro's PyQt5 is outdated but rather because sip is installed somewhat differently when not doing a `pip install` (I may be wrong). Just out of interest, does a direct `import sip` work for you?

In any case, a quick google search reveals that `from PyQt5 import sip` can fail on many systems. I have therefore rewritten the relevant function so that is does not rely on `sip` anymore. You can install the latest version from the development branch:

```
pip install git+https://github.com/samschott/maestral-dropbox.git@develop
```

**Comment by emansije** [comment]
> does a direct `import sip` work for you?

Yes, it does!

I've tried the development branch, as you suggest, and it's working fine, now. Thank you!

**Comment by SamSchott** [comment]
Ok, great. I will leave this issue open for now, just in case someone else encounters the same problem.

**Comment by SamSchott** [comment]
fixed in v0.5.0
